openedx/features/partners/admin.py

```diff
@@ -23,3 +23,18 @@ def partner_url(self, obj):
 """
 admin.site.register(PartnerUser)
 
+
+class PartnerCommunityModelAdmin(admin.ModelAdmin):
+    """
+    Django admin model to add category id to partner so that every user is added automatically to that category
+    """
+    list_display = ['id', 'partner', 'community_id']
+    search_fields = ('partner', 'community_id')
+
+    class Meta(object):
+        verbose_name = 'Partner Community'
+        verbose_name_plural = 'Partner Communities'
+
+
+admin.site.register(PartnerCommunity, PartnerCommunityModelAdmin)
```

Comment:
you can also use register decorator like
`@admin.register(PartnerCommunity) `
on class PartnerCommunityModelAdmin